packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -26,14 +26,18 @@ const Duration _kExpand = Duration(milliseconds: 200);
 /// the tile is expanded and collapsed: between [iconColor], [collapsedIconColor] and
 /// between [textColor] and [collapsedTextColor].
 ///
+/// The expansion arrow icon is shown on the right by default in left-to-right languages
```

Comment:
There are some unlikely-to-be-useful cases that we should probably assert about. Specifying a leading widget and controlAffinity=leading or likewise for trailing, seem likely to be mistakes.

Reply:
Specifying a leading widget and controlAffinity=leading allows replacing the default rotating icon with a custom icon. If controlAffinity=leading was not specified, there would be two icons because ExpansionTile would place the default rotating icon at the trailing edge.

Reply:
Oh, and there's also a test for overriding the icon like this.